akka-slf4j/src/test/scala/akka/event/slf4j/Slf4jLoggerSpec.scala

```diff
@@ -117,13 +122,24 @@ class Slf4jLoggerSpec extends AkkaSpec(Slf4jLoggerSpec.config) with BeforeAndAft
     "log info with slf4j marker" in {
       val slf4jMarker = MarkerFactory.getMarker("SLF")
       slf4jMarker.add(MarkerFactory.getMarker("ADDED")) // slf4j markers can have children
-      producer ! StringWithSlf4jMarkerMDC("security-wise interesting message", slf4jMarker)
+      producer ! StringWithSlf4jMarker("security-wise interesting message", slf4jMarker)
 
       awaitCond(outputString.contains("----"), 5 seconds)
       val s = outputString
       s should include("marker=[SLF [ ADDED ]]")
       s should include("msg=[security-wise interesting message]")
     }
+    "log info with slf4j marker and MDC" in {
+      val slf4jMarker = MarkerFactory.getMarker("SLF")
+      slf4jMarker.add(MarkerFactory.getMarker("ADDED")) // slf4j markers can have children
+      producer ! StringWithSlf4jMarkerMDC("security-wise interesting message", slf4jMarker, Map("ticketNumber" → 3671, "ticketDesc" → "Custom MDC Values"))
+
+      awaitCond(outputString.contains("----"), 5 seconds)
```

Comment:
`awaitAssert` is "always" better (error messages)
`awaitAssert(outputString should include ("----"))`